packages/plc/src/bin.ts

```diff
@@ -1,15 +1,15 @@
 import dotenv from 'dotenv'
+const env = process.env.ENV
```

Comment:
I honestly thought you weren't able to have any code in between static imports with ESM. Maybe our transpiler is able to handle it?

Regardless, its usually not considered best practice. if we want this code to run first, I'd drop it in its own `env.ts` file or similar

Reply:
:+1: added as a separate `src/env.ts` file for both PLC and PDS